### PR TITLE
[ARCTIC-622][Improvement]: fix file upload when use derby as storage

### PR DESCRIPTION
**Pull request by Aireed**
resolve https://github.com/NetEase/arctic/issues/622

## How was this patch tested?
- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [x] Run test locally before making a pull request

## Documentation

  - Does this pull request introduces a new feature? (no)
  - If yes, how is the feature documented? (not documented)

### CODE REVIEW

File: ams/ams-server/src/main/java/com/netease/arctic/ams/server/mapper/derby/DerbyPlatformFileInfoMapper.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  String TABLE_NAME = "platform_file_info";
+  /**
+   * add a file with content encoded by base64
+   */
+  @Insert("insert into " + TABLE_NAME + "(file_name,file_content_b64)" +
+          "values(#{fileInfo.fileName},#{fileInfo.fileContent})")
+  void addFile(@Param("fileInfo") PlatformFileInfo platformFileInfo);
+
+  // get file content encoded by base64 by fileId
+  @Select("select id from " + TABLE_NAME + " where file_content_b64=#{content}")
```
Comment (zhoujinsong) [inline ams/ams-server/src/main/java/com/netease/arctic/ams/server/mapper/derby/DerbyPlatformFileInfoMapper.java:24]: When there are multiple files with the same content，multiple `id` will be obtained.

**Review by zhoujinsong — COMMENTED** [review]
@Aireed I left some comments.

File: ams/ams-server/src/main/java/com/netease/arctic/ams/server/mapper/derby/DerbyPlatformFileInfoMapper.java (hunk 1 above)
Comment (Aireed) [reply ams/ams-server/src/main/java/com/netease/arctic/ams/server/mapper/derby/DerbyPlatformFileInfoMapper.java]: done， add limit 1 to keep only one result

**Review by zhoujinsong — APPROVED** [review]
LGTM.

**Pull request merged**